Vegetarian-Food-Finder/Api/ApiManger.swift

```diff
@@ -2,8 +2,35 @@
 //  ApiManger.swift
 //  Vegetarian-Food-Finder
 //
-//  Created by Abdullahi Ahmed on 7/6/22.
+//  Created by Abdullahi Ahmed on 7/7/22.
 //
 
 import UIKit
 
+final class ApiManger {
+    
+    private class ApiConstant {
+        static let ApiUrl = "https://api.yelp.com/v3/businesses/search?location=SF&categories=vegetarian"
```

Comment:
this URL is for searching a business, which will apply to only one of the methods of this API. I'd split it into the domain + version part ( `https://api.yelp.com/v3/`) and append the rest inside the method. That way you can reuse the same domain in multiple requests later.